src/commands/api/validate.js

```diff
@@ -0,0 +1,46 @@
+const BaseCommand = require('../../support/command/base-command')
+const { getApiIdentifierArg } = require('../../support/command/parse-input')
+const { getApi } = require('../../requests/api')
+const { pipeAsync } = require('../../utils/general')
+const { getResponseContent } = require('../../support/command/handle-response')
+
+class ValidateCommand extends BaseCommand {
+  async run() {
+    const { args } = this.parse(ValidateCommand)
+    const apiPath = getApiIdentifierArg(args)
+    const validationResult = await this.getValidationResult(apiPath)
+    // eslint-disable-next-line immutable/no-let
+    let hasCritical = false
+    const validationResultsStr = validationResult.validation
+          .map(err => {
+            if (err.severity === 'CRITICAL') hasCritical = true
+            return `${err.line}: \t${err.severity} \t${err.description}`
+          })
+          .join('\n')
+    if (validationResult.validation.length) this.log('line \tseverity \tdescription\n')
+    this.log(validationResultsStr)
```

Comment:
```suggestion
    if (validationResult.validation.length) {
      this.log('line \tseverity \tdescription\n')
      this.log(validationResultsStr)
    }
```

Reply:
This is not correct. But I add curly braces around the condition.

Reply:
Is `validationResultsStr` empty when `validationResult.validation.length === 0`?

Reply:
Yes, it is empty. But, this.log(validationResultsStr) logs an empty line so if want to keep that empty line it should be out of the condition block.